test/context.go

```diff
@@ -94,6 +94,24 @@ func (ctx *Context) AssertRegisterSpendNtfn(script []byte) {
 	}
 }
 
+// AssertTrackPayment asserts that a call was made to track payment, and
+// returns the track payment message so that it can be used to send updates
+// to the test.
+func (ctx *Context) AssertTrackPayment() TrackPaymentMessage {
+	ctx.T.Helper()
+
+	var msg TrackPaymentMessage
+	select {
+	case msg = <-ctx.Lnd.TrackPaymentChannel:
+
+	case <-time.After(Timeout):
```

Comment:
I think we generally use `defer timeout()()`, so this may not be necessary. Same for timeouts below.

Reply:
Following the assert pattern in this file, I don't think it hurts to have timeouts here in case we don't defer timeout somewhere.

Reply:
No, it doesn't hurt. Maybe something to revisit in the test refactor pr. Or not.